src/partition/repart.c

```diff
@@ -1668,6 +1669,33 @@ static int config_parse_make_dirs(
         }
 }
 
+static int config_parse_mkfs_options(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+                
+        _cleanup_strv_free_ char **l = NULL;
+        char ***opt = ASSERT_PTR(data);
+        const char *p = ASSERT_PTR(rvalue);
+        
+        if (p) {
+                l = strv_split(p, NULL);
```

Comment:
use strv_split_full() and pass at least EXTRACT_UNQUOTE and EXTRACT_CUNESCAPE.

Reply:
Not applicable here anymore but will keep it in mind.